docs/host-and-deploy/hosting-models.md

```diff
@@ -45,13 +45,27 @@ The downsides to the client-side hosting model are:
 * Larger download size and app load time
 * Less mature .NET runtime and tooling support (for example, limitations in .NET Standard support and debugging)
 
+> [!NOTE]
+> Visual Studio includes the **Blazor (ASP.NET Core hosted)** project template for creating a Blazor app that runs on WebAssembly and is hosted on an ASP.NET Core server. The server-side app merely serves the Blazor app to clients. The server-side app doesn't directly interact with the app after serving it to clients other than to possibly serve static assets from its web root (for example, *wwwroot*).
```

Comment:
server-side app -> ASP.NET Core app

Reply:
> The server-side app doesn't directly interact with the app after serving it to clients other than to possibly serve static assets from its web root (for example, *wwwroot*). 

You can have other interactions. For example, you might use a Web API or a SignalR Hub. 

How about something like this?:

Visual Studio includes the **Blazor (ASP.NET Core hosted)** project template for creating a Blazor app that runs on WebAssembly and is hosted on an ASP.NET Core server. The ASP.NET Core app handles serves the Blazor app to clients, but is otherwise a separate process. The client-side Blazor app can interact with the server over the network using web API calls or SignalR connections.